src/FSharp.NET.Sdk/FSharp.NET.Sdk.fsproj

```diff
@@ -0,0 +1,52 @@
+﻿<Project ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props" />
+  
+  <PropertyGroup>
+    <VersionPrefix>1.0.0-alpha</VersionPrefix>
+    <TargetFramework>netstandard1.3</TargetFramework>
+    <AssemblyName>FSharp.NET.Sdk</AssemblyName>
+    <PackageTags>f#;sdk</PackageTags>
+    <Authors>Enrico Sada; Jared Hester</Authors>
+    <PackageProjectUrl>https://github.com/dotnet/netcorecli-fsc</PackageProjectUrl>
+    <RepositoryType>git</RepositoryType>
+    <RepositoryUrl>https://github.com/dotnet/netcorecli-fsc</RepositoryUrl>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <Content Include="FSharp.NET.Sdk.targets">
+      <PackagePath>build\netstandard1.0</PackagePath>
+      <Pack>true</Pack>
+    </Content>
+    <Content Include="FSharp.NET.Core.Sdk.targets">
+      <PackagePath>build\netstandard1.0</PackagePath>
+      <Pack>true</Pack>
+    </Content>
+    <Content Include="FSharp.NET.Sdk.props">
+      <PackagePath>build\netstandard1.0</PackagePath>
+      <Pack>true</Pack>
+    </Content>
+    <Content Include="FSharp.NET.CrossTargeting.Sdk.targets">
+      <PackagePath>buildCrossTargeting</PackagePath>
+      <Pack>true</Pack>
+    </Content>
+  </ItemGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.NET.Sdk">
+      <Version>1.0.0-alpha-20161102-2</Version>
+      <PrivateAssets>All</PrivateAssets>
+    </PackageReference>
+    <PackageReference Include="NETStandard.Library">
+      <Version>1.6.0</Version>
+    </PackageReference>
+  </ItemGroup>
+
+
+  <ItemGroup>
```

Comment:
I needed to add that to prevent the dotnet cli` from failing when it tried to build (or maybe it was restore?) the project. iirc it'd give an error saying it didn't know what to do with the `fsproj` unless I had a reference to the `dotnet-compile-fsc`